### ISSUE TITLE
The name GeoSPARQL is misleading.

### ISSUE THREAD

**Issue by FransKnibbe** [opengeospatial/ogc-geosparql#156]
For me, an interesting conclusion from discussion at the recent GeoLD 2021 conference was that the message the name GeoSPARQL conveys could be misleading and limiting its uptake by a larger audience.
The name "GeoSPARQL" consists of two parts: "Geo" and "SPARQL". The first part, "Geo", seems to say that is about geography. But in truth GeoSPARQL can be used for other kinds of spatial data too. The second part "SPARQL", seems to say that it is an extension of SPARQL. But it is more than that: the ontology can be used to encode and publish spatial data on the web in without the use of SPARQL.

So people looking for a (web) ontology for spatial data on a casual glance might miss out on GeoSPARQL because of its name.

Changing GeoSPARQL's name at this moment will likely go too far. But perhaps its subtitle (A Geographic Query Language for RDF Data) can be changed to convey its more general purpose?

**Comment by nicholascar** [comment]
I agree that it it would be going too far to change it for this impending 1.1 release but I also agree that we should review the name immediately after this release and potentially change it for the next. This may mean moving directly to the hypothesized GeoSPARQL 2.0 equivalent, not an intermediary GeoSPARQL 1.2. This bigger leap seems to be gaining in favour.

**Comment by pietercolpaert** [comment]
You could also split the query language from the vocabulary, and give the vocabulary a more general name under the GeoSPARQL query language umbrella? The vocabulary could named something like the OGC “Spatial Vocabulary”

**Comment by nicholascar** [comment]
> You could also split the query language from the vocabulary

Well, something of a split  exists under the GeoSPARQL umbrella now - there are multiple resources with different roles in GeoSPARQL 1.1 - and we do think it's useful to have all these things linked, so we may yet have cause to have one unifying name. Perhaps something as generic as "OGC Semantic Web Spatial" within which is a "Spatial Ontology", various vocabularies, "SpatialSPARQL" etc.
